source/adapters/level_zero/command_buffer.cpp

```diff
@@ -180,16 +181,17 @@ ur_result_t calculateKernelWorkDimensions(
           --GroupSize[I];
         }
         if (GlobalWorkSize[I] / GroupSize[I] > UINT32_MAX) {
-          urPrint("urCommandBufferAppendKernelLaunchExp: can't find a WG size "
-                  "suitable for global work size > UINT32_MAX\n");
+          logger::debug(
+              "urCommandBufferAppendKernelLaunchExp: can't find a WG size "
+              "suitable for global work size > UINT32_MAX");
           return UR_RESULT_ERROR_INVALID_WORK_GROUP_SIZE;
         }
         WG[I] = GroupSize[I];
       }
-      urPrint(
-          "urCommandBufferAppendKernelLaunchExp: using computed WG size = {%d, "
-          "%d, %d}\n",
-          WG[0], WG[1], WG[2]);
+      logger::debug("urCommandBufferAppendKernelLaunchExp: using computed WG "
+                    "size = {{{}, "
+                    "{}, {}}}\n",
```

Comment:
Is the double newline (one explicit and one aded by the logger) intended?

Reply:
Done